pages/utils/graph_utils.py

```diff
@@ -1,7 +1,7 @@
 import datetime as dt
 
 # list of graph color hex
-color_seq = ["#f8dd70", "#c0bc5d", "#8e9b4c", "#62793d", "#3c582d", "#1c381d"]
+color_seq = ["#B5B682", "#c0bc5d", "#6C8975", "#485B4E", "#3c582d", "#376D39"]
```

Comment:
same ask of listing what these colors look like so we can edit them on-the-fly later.